lib/policies/proxy/index.js

```diff
@@ -1,3 +1,12 @@
 module.exports = {
-  policy: require('./proxy')
+  policy: require('./proxy'),
+  schema: {
+    type: 'object',
+    properties: {
+      serviceEndpoint: {type: 'string'},
```

Comment:
I understand that this is work in progress, just a reminder, pls add all options from corresponding middleware